pkg/apis/kubeflow.org/v1/tensorflow_types.go

```diff
@@ -122,9 +122,14 @@ const (
 	TFJobReplicaTypeEval ReplicaType = "Evaluator"
 )
 
+// IsChiefOrMaster returns true if the type is Master or Chief.
+func IsChiefOrMaster(typ ReplicaType) bool {
+	return typ == TFJobReplicaTypeChief || typ == TFJobReplicaTypeMaster
+}
+
```

Comment:
@tenzen-y Does it make sense to move this function in `webhooks/tensorflow/tfjob_webhook.go` since we use it there and it will help us to avoid creation of additional file named `tensorflow_types_test.go`?

Reply:
Indeed. I'm wondering if we can put this function in the webhook package.
However, I decided to put this here since this function is not only for the webhooks like this: https://github.com/search?q=repo%3Akubeflow%2Ftraining-operator%20IsChiefOrMaster&type=code

Reply:
Yes, we need it since the webhook verifies only if the master/chief role replica is 0 or 1.
When TFJob doesn't have any master/chief role, the training operator switches there based on the check.

Reply:
Alright, since we need additional check to verify if `rtype` is Chief of Master